package.json

```diff
@@ -123,6 +124,7 @@
     "babel-plugin-transform-import-meta": "^2.1.0",
     "eslint": "^8.9.0",
     "eslint-config-prettier": "^8.4.0",
+    "eslint-plugin-react-hooks": "^4.3.0",
```

Comment:
The version of this plugin that the preset we are using is out of date and is causing conflicts with my editor